src/test/java/com/devcourse/voucherapp/entity/MenuTest.java

```diff
@@ -0,0 +1,31 @@
+package com.devcourse.voucherapp.entity;
+
+import static com.devcourse.voucherapp.entity.Menu.getMenu;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.devcourse.voucherapp.exception.MenuInputException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class MenuTest {
+
+    @DisplayName("존재하는 메뉴를 입력했을 때, 해당 메뉴 객체가 반환되는지 테스트")
```

Comment:
테스트의 네이밍은 `상황`,`실행`,`기대값`의 조합으로 가급적 문장형이 되는게 좋습니다.
```suggestion
    @DisplayName("존재하는 메뉴를 입력했을 때, 해당 메뉴 객체가 반환된다.")
```